tests/rsa/mod.rs

```diff
@@ -39,6 +39,19 @@ fn round_trip_sign_verification_pem_pkcs1() {
     }
 }
 
+#[test]
+fn round_trip_sign_verification_certificate_pem_pkcs1() {
```

Comment:
Is this a `pkcs#1` certificate? I couldn't actually find any example of a pkcs#1 rsa certificate but I would have guessed it would have an envelope like `-----BEGIN RSA CERTIFICATE-----` ?

Since the envelop header doesn't have to explicitly specify that it's an RSA certificate I think that implies that it's `pkcs#8` or some other later pkcs standard.

It seems like we can just keep the "pkcs8" variant or perhaps even just refer to these certificates as being in "pem" format?

Reply:
There is no pkcs#1 or pkcs#8 formats for certificates, all possible tags supported by opessl are listed [here](https://github.com/openssl/openssl/blob/master/include/openssl/pem.h#L35-L60).  So, yes, it is just a "pem" format.